src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java

```diff
@@ -302,33 +310,50 @@ public long getTaskCount(SearchTaskRequest searchTaskRequest) {
             if (!assigneeTaskList.isEmpty()) {
                 searchResults = assigneeTaskList;
             }
+            final List<Task> taskList = performSearchAction(searchResults, accessControlResponse, permissionsRequired);
 
-            return performSearchAction(searchResults, accessControlResponse, permissionsRequired);
+            if (taskList.isEmpty()) {
```

Comment:
Can we add a unit test to cover this line? Sonar has flagged it. Whilst it may be covered in ft, Its also essential to have unit test coverage as we maintain the bottom up approach for testing standard